changelog.d/1-api-changes/pr-3213

```diff
@@ -0,0 +1 @@
+Non-binding team endpoints are removed from API version V4
```

Comment:
```suggestion
Remove deprecated non-binding team endpoints from API version V4
```

Reply:
Since this appears in the changelog, it is already in the past. So I prefer past tense or even better "have been removed" (present perfect progressive). But I will not hold my opinion very strongly. :)